app/Models/Device.php

```diff
@@ -15,7 +15,7 @@
 use LibreNMS\Util\Url;
 use LibreNMS\Util\Time;
 
-class Device extends BaseModel
+class Device extends DeviceRelatedModel
```

Comment:
@murrant This is up to you, a bit quirky, but it saves having to duplicate functions from `DeviceRelatedModel.php`

Reply:
I think I prefer the duplicated function...  It doesn't contain any code so the duplication is minor.